Libraries/Utilities/Platform.ios.js

```diff
@@ -12,8 +12,21 @@
 
 'use strict';
 
+var invariant = require('invariant');
+
 var Platform = {
   OS: 'ios',
+  select: (obj: Object) => {
+    invariant(
+      obj && typeof obj === 'object',
+      'Platform.select: Must be called with an object'
+    );
+    invariant(
```

Comment:
Think we don't want this because you should be able to run code like this:
```js
Platform.select({
  android: { ... }
})
```
and have the Android style get filtered out on iOS even if there are no iOS-specific styles.